### PR TITLE
[SEMVER-MAJOR] update copyright notices and license

### PR DESCRIPTION
**Pull request by loay**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,25 +1,210 @@
-Copyright (c) IBM Corp. 2016. All Rights Reserved.
+Copyright (c) IBM Corp. 2014,2016. All Rights Reserved.
```
Comment (bajtos) [inline LICENSE.md]: This code was written in 2016, why should the copyright say 2014?

File: LICENSE.md [hunk 2]
```diff
@@ -1,25 +1,210 @@
...
 Node module: strong-error-handler
-This project is licensed under the MIT License, full text below.
+This project is licensed under the Artistic License 2.0, full text below.
```
Comment (bajtos) [inline LICENSE.md]: What is the rationale for changing the license from MIT to AL-2.0? I believe most loopback modules use MIT.

**Comment by bajtos** [comment]
@rmg PTAL too

**Comment by rmg** [comment]
What is the reason for changing the license? It seems unnecessary to me.

**Comment by loay** [comment]
I thought we had to change it to A-L !
Closing this PR then since it is unnecessary.

**Pull request closed without merging**